internal/animation/runner.go

```diff
@@ -23,9 +24,19 @@ func (r *Runner) Start(a *fyne.Animation) {
 
 	if !r.runnerStarted {
 		r.runnerStarted = true
+		if r.animations == nil {
```

Comment:
I think it could avoid duplication if moved up 4 lines? (Before the outer if)

Reply:
One is for initializing `r.animations` and the other for `r.pendingAnimations` though. I could create a helper function if you think it's cleaner, invoked like `r.animations = initIfNil(r.animations)`?

Reply:
I think both will be fine. So please choose the one you prefer.

I just raised a point it could have been moved out

Another solution is a function where you pass a 

```
func initL(l *[]anim) {
if l == nil {
l= make....
}
}
```

Then you call initL(r.animations) then same for r.pendingAnimations

But maybe I missed something

Reply:
I think it is fine as it is. It is not much in the way of duplication

Reply:
Sorry I missed that it was not identical